CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 3.2.0 (April 22, 2022)
```

Comment:
Nit: (as discussed in slack), I'd go for 3.1.3 because it doesn't contain new features. But it's quite optional.

Reply:
Agreed. Fixed.